Dockerfile

```diff
@@ -29,13 +29,13 @@ ADD perseo-utils /code/perseo-utils/
 ADD perseo_core-entrypoint.sh /code
 
 RUN apt-get update && \
-    apt-get install -y maven openjdk-8-jdk && \
+    apt-get install -y maven openjdk-11-jdk && \
```

Comment:
Should we remove Java 8, 9 and 10 in .travis.yml ? Or they have still making sense in some environment and it is a good thing to regression on them?

Reply:
Not sure about this, it would be nice to know if at any time perseo is no longer supported by those jdks, don't you thing?

Reply:
Ok. Let's kept them by the moment. It doesn't have any cost :)

NTC